mix.exs

```diff
@@ -22,6 +22,7 @@ defmodule SituationRoom.MixProject do
   defp deps do
     [
       {:plug, "~> 1.13.5"},
+      {:ecto, "~> 3.7.2"},
```

Comment:
Squash this commit back into the first commit since that is where this line was originally removed